data/json/npcs/robofac/intercom/robofac_intercom_prototypes.json

```diff
@@ -302,6 +302,11 @@
       }
     ]
   },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_ARMOR_BREAKTHROUGH_END",
+    "type": "talk_topic",
+    "dynamic_line": "Thank you.  The mantle and skirt will clip to your MDS, and the other pieces can be worn standalone.  If you happen upon anything else like this in the future, we would be happy to collaborate again."
```

Comment:
This is a great addition, and it looks like it goes to here in all the right places 👍